<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCBConf</class>
 <widget class="QDialog" name="GCBConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>配置端口映射文件</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="impIU_btn">
           <property name="text">
            <string>导入智能设备配置文件</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table_IU">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <column>
          <property name="text">
           <string>选择</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>数据集名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>目标地址</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>    端口    </string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>转发端口</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>源地址</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="impMU_btn">
           <property name="text">
            <string>导入合并单元配置文件</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table_MU">
         <column>
          <property name="text">
           <string>选择</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>数据集名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>目标地址</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>    端口    </string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>转发端口</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>源地址</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_genIUGS">
       <property name="text">
        <string>生成智能终端gs.xml</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_genMUGS">
       <property name="text">
        <string>生成合并单元gs.xml</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btn_cancel">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
